crates/bevy_utils/src/lib.rs

```diff
@@ -32,10 +32,10 @@ impl std::hash::BuildHasher for FixedState {
     }
 }
 
-/// A [`HashMap`][std::collections::HashMap] implementing [aHash][aHash], a high
+/// A [`HashMap`][std::collections::HashMap] implementing [`aHash`], a high
```

Comment:
does this link still works? I think cargo-doc would try to render it as an indoc link rather than the external link referenced by `[aHash]`

Reply:
Yup! I double checked, it still works. If the link text itself is identical to the link specifier you don't have to write it twice.

Reply:
Also we don't have a struct named `aHash` for it to conflict with.